code/src/components/Age.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export const Age = ({ name, setAge, increase, decrease }) => {
+
+    return (
+    <>
+     <div className="container">
+      <h1>Hello {name}! Select your age:</h1>
+      <div className="checkboxes">
```

Comment:
A small definition detail: I think these are called radio buttons. Checkboxes are for multiple options.